pyanaconda/ui/gui/spokes/keyboard.py

```diff
@@ -502,6 +503,10 @@ def _refresh_switching_info(self):
                                                "supported when using RDP.\n"
                                                "However the settings will be used "
                                                "after the installation."))
+        elif not conf.system.supports_compositor_keyboard_layout_shortcut:
+            self._layoutSwitchLabel.set_text(_("Keyboard layouts switching by "
+                                               "keyboard shortcuts is not supported "
```

Comment:
I'd say is "not currently supported", but I don't really know, this would need en eye of someone with good english so feel free to ignore...